com.github.neithern.g4music.appdata.xml.in

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is never referenced or installed. also `.in` ?

And it is way insufficient. missing `release`, `screenshot`, `url` and probably a few other I forget.